src/main/java/frc/robot/end_effector/EndEffectorIOVortex.java

```diff
@@ -0,0 +1,96 @@
+    private final RelativeEncoder coralIntakeEncoder = coralIntake.getEncoder();
+
+    private final SparkFlex algaeIntake = new SparkFlex(EndEffectorConstants.ALGAE_INTAKE_CAN, MotorType.kBrushless);
+    private final RelativeEncoder algaeIntakeEncoder = algaeIntake.getEncoder();
+
+    private final SparkFlex algaeWrist = new SparkFlex(EndEffectorConstants.ALGAE_WRIST_CAN, MotorType.kBrushless);
+    private final RelativeEncoder algaeWristEncoder = algaeWrist.getEncoder();
+
+    private final DigitalInput coralIntakeBeamBreakDigitalInput = new DigitalInput(0); //configure the DIO port for the Beam Break Sensor after the robot is wired.
+    private final LaserCan LaserCan = new LaserCan(0);
+
+    public EndEffectorIOVortex () {
+
+        SparkFlexConfig config = new SparkFlexConfig();
+
+        config.idleMode(IdleMode.kBrake).smartCurrentLimit(EndEffectorConstants.CORAL_INTAKE_CURRENT_LIMIT).voltageCompensation(12.0);
+        config.idleMode(IdleMode.kBrake).smartCurrentLimit(EndEffectorConstants.ALGAE_INTAKE_CURRENT_LIMIT).voltageCompensation(12.0);
+        config.idleMode(IdleMode.kBrake).smartCurrentLimit(EndEffectorConstants.ALGAE_WRIST_CURRENT_LIMIT).voltageCompensation(12.0);
+
+        config.encoder
+            .positionConversionFactor(2.0 * Math.PI)
+            .velocityConversionFactor((2.0 * Math.PI) / 60.0)
+            .uvwMeasurementPeriod(10)
+            .uvwAverageDepth(2);
+
+        coralIntake.configure(config, ResetMode.kResetSafeParameters, PersistMode.kPersistParameters);
+        algaeIntake.configure(config, ResetMode.kResetSafeParameters, PersistMode.kPersistParameters);
+        algaeWrist.configure(config, ResetMode.kResetSafeParameters, PersistMode.kPersistParameters);
+    }
+
+    @Override
+    public void updateInputs (EndEffectorIOInputs inputs) {
+
+        inputs.coralIntakePosition = coralIntakeEncoder.getPosition();
+        inputs.coralIntakeVelocity = coralIntakeEncoder.getVelocity();
+        inputs.coralIntakeVoltage = coralIntake.getAppliedOutput() * coralIntake.getBusVoltage();
+        inputs.coralIntakeCurrent = coralIntake.getOutputCurrent();
+
+        inputs.algaeIntakePosition = algaeIntakeEncoder.getPosition();
+        inputs.algaeIntakeVelocity = algaeIntakeEncoder.getVelocity();
+        inputs.algaeIntakeVoltage = algaeIntake.getAppliedOutput() * algaeIntake.getBusVoltage();
+        inputs.algaeIntakeCurrent = algaeIntake.getOutputCurrent();
+        
+        inputs.algaeWristPosition = algaeWristEncoder.getPosition();
+        inputs.algaeWristVelocity = algaeWristEncoder.getVelocity();
+        inputs.algaeWristVoltage = algaeWrist.getAppliedOutput() * algaeWrist.getBusVoltage();
+        inputs.algaeWristCurrent = algaeWrist.getOutputCurrent();
+
+        inputs.coralIntakeBeamBreak = coralIntakeBeamBreakDigitalInput.get();
+    }
+   
+    public void periodic() {
+
+        LaserCan.Measurement measurement = LaserCan.getMeasurement();
```

Comment:
Instead of printing the LaserCAN's measurement to the console, you need to update the IOInputs in `updateInputs()`.